Source/WebGPU/WebGPU/Buffer.mm

```diff
@@ -389,3 +389,9 @@ void wgpuBufferSetLabel(WGPUBuffer buffer, const char* label)
 {
     WebGPU::fromAPI(buffer).setLabel(WebGPU::fromAPI(label));
 }
+
+WGPUBufferUsage wgpuBufferGetUsage(WGPUBuffer buffer)
+{
+    // FIXME: this shouldn't need a cast - https://github.com/webgpu-native/webgpu-headers/issues/172
+    return static_cast<WGPUBufferUsage>(WebGPU::fromAPI(buffer).usage());
```

Comment:
should we make fromAPI() with a bunch of overloads for these?

Reply:
Maybe I misunderstand the suggestion but I don't know if we can, since the argument would be a uint32_t (WGPUBufferUsageFlags is typedef'ed as uint32_t)

Reply:
awww bummer. ok